core/src/components/input/test/masking/input.e2e.ts

```diff
@@ -9,11 +9,13 @@ test.describe('input: masking', () => {
   test('should filter out spaces', async ({ page, skip }) => {
     skip.rtl();
 
+    const ionInput = await page.spyOnEvent('ionInput');
     const input = page.locator('#inputTrimmed');
 
     await input.click();
 
     await page.keyboard.type('S p a c e s');
+    await ionInput.next();
```

Comment:
Since `ionInput` fires on each individual key press, I wonder if this could sometimes only catch the first n number of characters? Maybe blurring the input and watching for `ionChange` would be better?

Reply:
This is going into `main`, so `ionChange` is not getting fired on `blur` yet. However, I changed the test so it waits for all `ionInput` events to be received:  `await expect(ionInput).toHaveReceivedEventTimes(11);`.